RNA-seq/02-gastric_cancer_tximeta.Rmd

```diff
@@ -0,0 +1,200 @@
+---
+title: "Gastric cancer: gene-level summarization with `tximport`"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2021**
+
+In this notebook, we'll import transcript level output from `salmon quant` and
+summarize it to the gene level using 'transcript expression import' - [`tximport`](https://bioconductor.org/packages/release/bioc/html/tximport.html) and 'tximport meta' - [`tximeta`](https://bioconductor.org/packages/release/bioc/html/tximeta.html). 
```

Comment:
> Only making this comment here, though it applies throughout. I think we were starting to standardize on one sentence per line, for easier tracking of revisions & more consistent line numbers. (If we were to do more aggressive line breaks, this would need to be broken up, but I would just make this one line.)

My impulse was to do this but then I thought I remembered our standard for this repo was different. Is this something that can be added to `CONTRIBUTING.md`? Also I think the spacing/line stuff is weird in a lot of places, so if there's a way we can fix it all at once that's probably good. I'll change it in this notebook to start.

Reply:
I think it was different for a while, but we were starting to standardize as notebooks got modified. Pretty sure we changed everything in scRNA-seq, but possibly not.